docs/mvvm/MigratingFromMvvmBasic.md

```diff
@@ -0,0 +1,159 @@
+
+This article focuses exclusively on migration and does not cover how to use the additional functionality that the library provides.
+
+## Installing the WCT MVVM Toolkit
+
+To use the Windows Community Toolkit MVVM framework, you must install the NuGet package into your existing application.
+
+### Install via .NET CLI
+
+```
+dotnet add package Microsoft.Toolkit.Mvvm --version x.x.x
+```
+
+### Install via PackageReference
+
+```
+<PackageReference Include="Microsoft.Toolkit.Mvvm" Version="x.x.x" />
+```
+
+## Updating a project
+
+There are four steps to migrate the code generated by Windows Template Studio.
+
+1. Delete old files.
+2. Replace use of `Observable`.
+3. Add new namespace references.
+4. Update methods with different names.
+
+### 1. Delete old files
+
+MVVM Basic is comprised of two files
+
+```csharp
+\Helpers\Observable.cs
+\Helpers\RelayCommand.cs
+```
+```vb
+\Helpers\Observable.vb
+\Helpers\RelayCommand.vb
+```
+
+Delete both of these files.
+
+If you try and build the project at this point you will see lots of errors. These can be useful for identifying files that require changes.
+
+### 2. Replace Use of `Observable`
+
+The `Observable` class was used as a base class for ViewModels. The MVVM Toolkit contains a similar class with additional functionality that is called `ObservableObject`.
+
+Change all classes that previously inherited from `Observable` to inherit from `ObservableObject`.
+
+For example
+
+```csharp
+    public class MainViewModel : Observable
+```
+```vb
+    Public Class MainViewModel
+        Inherits Observable
+```
+
+will become
+
+```csharp
+    public class MainViewModel : ObservableObject
+```
+```vb
+    Public Class MainViewModel
+        Inherits ObservableObject
+```
+
+### 3. Add new namespace references
+
+Add a reference to the `Microsoft.Toolkit.Mvvm.ComponentMode` namespace in all files where there is a reference to `ObservableObject`.
+
+You can either add the appropriate directive manually, of move the cursor to the `ObservableObject` and press `Ctrl+.` to access the Quick Action menu to add this for you.
+
+```csharp
+using Microsoft.Toolkit.Mvvm.ComponentModel;
+```
+```vb
+Imports Microsoft.Toolkit.Mvvm.ComponentModel
+```
+
+Add a reference to the `Microsoft.Toolkit.Mvvm.Input` namespace in all files where there is a reference to `RelayCommand`.
+
+You can either add the appropriate directive manually, of move the cursor to the `RelayCommand` and press `Ctrl+.` to access the Quick Action menu to add this for you.
+
+```csharp
+using Microsoft.Toolkit.Mvvm.Input;
+```
+```vb
+Imports Microsoft.Toolkit.Mvvm.Input
+```
+
+### 4. Update methods with different names
+
+There are two methods that must be updated to allow for different names for the same functionality.
+
+All calls to `Observable.Set` must be replaced with calls to `ObservableObject.SetProperty`.
+
+So,
+
+```csharp
+    set { Set(ref _elementTheme, value); }
```

Comment:
Super minor nit: no property expression?
As in:
```csharp
set => Set(ref _elementTheme, value);
```

Reply:
We use the expanded (with curly  brackets) versions of getters and setters everywhere in the generated code as a way of ensuring consistency. The aim is to avoid anyone new to coding (a key target audience of WinTS) from needing to wonder about why different syntax is used in different places.

Reply:
Oh I see, makes sense! Thanks 😄